sdk/keyvault/keyvault-admin/src/generated/keyVaultClient.ts

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as coreHttp from "@azure/core-http";
+import * as operations from "./operations";
+import * as Parameters from "./models/parameters";
+import * as Models from "./models";
+import * as Mappers from "./models/mappers";
+import { KeyVaultClientContext } from "./keyVaultClientContext";
+import {
+  KeyVaultClientOptionalParams,
+  KeyVaultClientFullBackupOptionalParams,
```

Comment:
@christothes was the PR to consolidate params and responses for full and selective backup not merged yet? Ideally, these are the same input and output models since the structures are exactly the same.

Reply:
No, it’s still active https://github.com/Azure/azure-rest-api-specs/pull/9842